docs/source/data-proc-additional.ipynb

```diff
@@ -54,7 +54,68 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "<!-- ### ADD FIGURE -->"
+        "(data-proc:z-variables)=\n",
+        "## Vertical coordinate (_z_-axis) variables"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The SONAR-netCDF4 v1 convention defines several variables in the `Platform` group that are referenced to a common Platform Coordinate System (PCS) vertical (_z_-axis) origin, ${PCS}_o$. These variables, when available, enable the calculation of the depth of each echo sample from the water surface. These variables are listed below together with their definitions, with edits when appropriate."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "The _z_ axis is positive-down from the platform origin ${PCS}_o$, and all variables except for `vertical_offset` are defined with respect to ${PCS}_o$. All variables are in units meters.\n",
+        "\n",
+        "- `water_level`: Distance from the ${PCS}_o$ to the *nominal* (fixed) water level measured along the _z_-axis of the platform coordinate system (positive values are below the origin).\n",
+        "- `vertical_offset`: Time-varying distance from the *nominal* water level to the *actual* water level measured along the _z_-axis of the platform coordinate system (*positive values are when the actual water level is below the nominal water level*). For ships and similar platforms, this is *heave*, but the concept applies equally well to underwater vehicle depth.\n",
+        "- `MRU_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the motion reference unit (MRU) sensor origin.\n",
+        "- `position_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the latitude/longitude sensor origin.\n",
+        "- `transducer_offset_z`: Distance along the _z_-axis from ${PCS}_o$ to the sonar transducer.\n",
+        "\n",
+        "Using these variables, the depth of the transducer with respect to the actual water level can be calculated as:\n",
+        "\n",
+        "```\n",
+        "transducer_depth = transducer_offset_z - water_level - vertical_offset\n",
+        "```\n",
+        "\n",
+        "and the depth for a given echo sample is:\n",
+        "\n",
+        "```\n",
+        "sample_depth = transducer_depth + echo_range_z\n",
+        "```\n",
+        "\n",
+        "where `echo_range_z` is the _z_-axis component (positive down) of `echo_range`."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When the transducer points vertically (up or down), `echo_range_z = echo_range`. When the transducer is mounted with an angle to the vertical axis, the beam direction vector (defined by the variables `beam_direction_x`, `beam_direction_y`, `beam_direction_z` in the `Sonar/Beam_groupX`) should be used together with `echo_range` to calculate `echo_range_z`."
```

Comment:
As `echo_range_z` needs to follow the positive-down convention, I think an edit along the lines of what I've suggested would need to be made:

```suggestion
        "When the transducer points vertically, `echo_range_z = echo_range` (transducer points down) or `echo_range_z = -echo_range` (transducer points up). When the transducer is mounted with an angle to the vertical axis, the beam direction vector (defined by the variables `beam_direction_x`, `beam_direction_y`, `beam_direction_z` in the `Sonar/Beam_groupX` group) should be used together with `echo_range` to calculate `echo_range_z`."
```

Reply:
This sounds good to me. To try to not let the user to be even more confused (or actually doing the opposite?), how about adding a short leading in in the beginning of the sentence "To compute the sample depth, when the transducer points ..."?
```suggestion
        "To compute the sample depth, when the transducer points vertically, `echo_range_z = echo_range` (transducer points down) or `echo_range_z = -echo_range` (transducer points up). When the transducer is mounted with an angle to the vertical axis, the beam direction vector (defined by the variables `beam_direction_x`, `beam_direction_y`, `beam_direction_z` in the `Sonar/Beam_groupX`) should be used together with `echo_range` to calculate `echo_range_z`."
```

Reply:
I like your version. I'll commit it